### PR TITLE
[#6425 A11Y] lighthouse fixes

### PR DESCRIPTION
**Pull request by phillimorland**

### CODE REVIEW

File: meinberlin/apps/budgeting/templates/meinberlin_budgeting/proposal_list.html [hunk 1]
```diff
@@ -27,6 +27,7 @@
                     <div class="form-group">
                         {% csrf_token %}
                         <div class="widget--{{ token_form.token|widget_type }}">
+                            <label class="visually-hidden-focusable">{% trans 'Voting token form fields' %}</label>
```
[the remark this answers is not in the document]
Comment (phillimorland) [reply meinberlin/apps/budgeting/templates/meinberlin_budgeting/proposal_list.html:30]: after some research I think we might need to refactor this and do it like in the design with 1 input but number sets separated by a space, i will look into this now

File: meinberlin/apps/votes/forms.py [hunk 2]
```diff
@@ -1,34 +1,62 @@
...
+            raise forms.ValidationError(self.error_messages['invalid'])
+
+        return value
+
+    def token_from_number(self, num):
+        # find this token, based on the token number
+        for token in self.token:
+            for pattern in token['patterns']:
+                if (str(pattern) == str(num)[:len(str(pattern))]):
+                    return token
```
Comment (phillimorland) [inline meinberlin/apps/votes/forms.py:54]: @fuzzylogic2000 here it is if you want to take a look, I will keep poking till lunch then maybe go on to react update test branch

Comment (fuzzylogic2000) [reply meinberlin/apps/votes/forms.py]: Right, I can also fix the (backend) test! Maybe when we are done with everything here?

File: meinberlin/apps/votes/assets/token_input.js [hunk 3]
```diff
@@ -0,0 +1,109 @@
+function init () {
```
Comment (phillimorland) [inline meinberlin/apps/votes/assets/token_input.js:1]: @goapunk @khamui so I got this from a stackoverflow suggestion on how to autoformat a credit card entry, I have kind of cleaned it but not enough but before i spend more time I wanted to sanity check with you guys?

**Pull request closed without merging**